### PR TITLE
Train

### PR DESCRIPTION
**Pull request by kirsten-stallings**
Added logistice regression code. Pictures work now

### CODE REVIEW

File: ML_tutorial.ipynb [hunk 1]
```diff
@@ -4012,370 +4113,2223 @@
...
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>3 rows × 297 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   N_serie  US viewers (million)  Runtime (mins)  IMDB votes  Imdb Rating  \\\n",
+       "0      0.0              0.002026             0.4    0.078472          0.5   \n",
+       "1      0.0              0.002026             0.4    0.078472          0.5   \n",
+
```
Comment (jambran) [inline ML_tutorial.ipynb]: there's a gif I use in the `word_embedding_tutorial.ipynb`that compares the loss nicely- feel free to use it here too. It is on a neural net though, not LR :/

Comment (jambran) [inline ML_tutorial.ipynb]: maybe "change the weights in the model" instead of punish

Comment (jambran) [inline ML_tutorial.ipynb]: You're last sentence could really be its own section. How do I know when to stop training? 
1. fixed number of training epochs
2. use a val set to know when the model starts overfitting
3. check the gradient to see if we're still making significant weight updates

I don't remember ever using a cost threshold, as it feels like a rectal database number, but let me know if I'm missing something.

Comment (jambran) [inline ML_tutorial.ipynb]: I would avoid using the word "features" here 😅 

Maybe knobs and dials, settings, something else.

Comment (jambran) [inline ML_tutorial.ipynb]: yeah, so this is using option 3 for "when to stop training"

Comment (jambran) [inline ML_tutorial.ipynb]: Yesssssssssssss this makes me so happy

Comment (jambran) [inline ML_tutorial.ipynb]: missing paren...? Did git screw up? Or jupyter notebooks? Can't tell

[the remark this answers is not in the document]
Comment (kirsten-stallings) [reply ML_tutorial.ipynb]: love it 😍

Comment (kirsten-stallings) [reply ML_tutorial.ipynb]: good call. fixed!

Comment (kirsten-stallings) [reply ML_tutorial.ipynb]: git's fault!
